### PR TITLE
UIQM-740 Don't show warn/fail error toasts when there are no warns/fails.

### PR DESCRIPTION
**Pull request by BogdanDenis**
## Description
Don't show warn/fail error toasts when there are no warns/fails.

## Screenshots

https://github.com/user-attachments/assets/7b50e81e-23ba-4b43-b3fd-e0d80819a114

## Issues
[UIQM-740](https://folio-org.atlassian.net/browse/UIQM-740)

### CODE REVIEW

File: src/QuickMarcEditor/QuickMarcEditor.test.js [hunk 1]
```diff
@@ -895,6 +895,30 @@ describe('Given QuickMarcEditor', () => {
     });
   });
 
+  describe('when saving form without validation warnings or errors', () => {
```
Comment (Dmytro-Melnyshyn) [inline src/QuickMarcEditor/QuickMarcEditor.test.js:898]:
```suggestion
  describe('when saving form without validation warnings and errors', () => {
```

File: src/QuickMarcEditor/QuickMarcEditor.test.js [hunk 2]
```diff
@@ -895,6 +895,30 @@ describe('Given QuickMarcEditor', () => {
...
+        getByTestId,
+      } = renderQuickMarcEditor();
+
+      const contentField = getByTestId('content-field-3');
+
+      fireEvent.change(contentField, { target: { value: '' } });
+      await fireEvent.click(getByText('stripes-acq-components.FormFooter.save'));
+    });
+
+    it('should show a toast notification about validation warning and error', async () => {
```
Comment (Dmytro-Melnyshyn) [inline src/QuickMarcEditor/QuickMarcEditor.test.js:913]:
```suggestion
    it('should not show a toast notification about validation warning and error', async () => {
```

File: src/QuickMarcEditor/QuickMarcEditor.test.js (hunk 1 above)
Comment (BogdanDenis) [reply src/QuickMarcEditor/QuickMarcEditor.test.js:898]: In English you can say "without X or Y" meaning "without X and without Y". A little weird, but that's how it is.
Interestingly, it works the same way as De Morgan's law in boolean logic, where "not (A or B) = (not A) and (not B)"

Comment (zburke) [reply src/QuickMarcEditor/QuickMarcEditor.test.js:898]: Wow, I haven't thought about De Morgan since grad school!

**Review by zburke — APPROVED** [review]

**Review by Dmytro-Melnyshyn — APPROVED** [review]

**Pull request merged**